kahuna/public/js/components/gr-batch-export-original-images/gr-batch-export-original-images.js

```diff
@@ -9,6 +9,16 @@ batchExportOriginalImages.controller('grBatchExportOriginalImagesCtrl', [
     function($scope, $rootScope, $state, mediaCropper) {
         let ctrl = this;
 
+        const checkForFullCrops = () => ctrl.images.every(
+            image => image.data.exports.some(
```

Comment:
Not blocking getting this out as we can raise in a further PR, however, should we push this into the [imageAccessor](https://github.com/guardian/grid/blob/master/kahuna/public/js/services/image-accessor.js) to keep knowledge of the shape of the API in one place?

Reply:
Yes we should, I've made a card for it.